### DISCUSSION TITLE
Request for Project Setup Guidance in Replit

### DISCUSSION THREAD

**Discussion by marekrakowski** [AntonOsika/gpt-engineer#200] [Q&A]
Hi,

I am quite interested in this project and I'm eager to set it up in a Replit environment. I noticed that a user previously asked for setup instructions specific to a Windows environment without WSL (https://github.com/AntonOsika/gpt-engineer/discussions/187). Similar to them, I also find the instructions provided in the "Setup" section in README a bit challenging to follow, primarily due to my basic understanding of development.

I would be really grateful if someone could provide a step-by-step guide to configuring the project in Replit. OpenAI-based solutions are of great interest to me and I'd love to learn more by getting this project up and running. I kindly ask for your patience and understanding as I navigate through this process.

Thanks in advance for your help.

**Comment by marekrakowski** [comment]
I just discovered how to look for closed topis and found this discussion https://github.com/AntonOsika/gpt-engineer/discussions/108
However, the reply is still not very clear for me. Of course I am doing my own research and googling but still I don't fully understand the instructions ;/

**Comment by jewilhel** [comment]
I really need this as well!
